tests/functional/a/arguments_renamed.py

```diff
@@ -0,0 +1,43 @@
+# pylint: disable=arguments-differ, missing-docstring, no-self-use, too-few-public-methods, useless-object-inheritance, line-too-long
+"""Test where we are emitting arguments-renamed when the
+    arguments have been renamed without changing type.
+"""
+
+class Parent(object):
+
+    def test(self, arg: int):
+        return arg + 1
+
+
+class Child(Parent):
+
+    def test(self, arg1: int): # [arguments-renamed]
+        return arg1 + 1
+
+class Child2(Parent):
+
+    def test(self, var: int): # [arguments-renamed]
+        return var + 1
+
+class ParentDefaults(object):
+
+    def test1(self, arg: str, barg: str):
+        print(f"Argument values are {arg} and {barg}")
+
+    def test2(self, arg: str, barg: str):
+        print(f"Argument values are {arg} and {barg}!")
+
+    def test3(self, arg1: str, arg2: bool):
+        print(f"arguments: {arg1} {arg2}")
+
+class ChildDefaults(ParentDefaults):
+
+    def test1(self, param1: str, param2: str): # [arguments-renamed]
```

Comment:
```suggestion
    def test1(self, param1: str, param2: str): # [arguments-renamed, argument-renamed]
```
I think the expected result is two messages. One with `arg` renamed to `param1` and one with `barg` renamed to `param2` in the message. What do you think ?

Reply:
Since the output message is "Parameters have been renamed" and not "Parameter" and also it doesn't specify the arguments, so I believe one message is enough.